### ISSUE TITLE
Using postgis-jdbc-jts and reading POLYGON EMPTY from ResultSet throws exception

### ISSUE THREAD

**Issue by lukaszm89** [postgis/postgis-java#118]
When using JTS driver version, if ResultsSet contains `POLYGON EMPTY` library is throwing Exception:
```
java.sql.SQLException: Error parsing SQL data:java.lang.NegativeArraySizeException: -1
	at net.postgis.jdbc.jts.JtsGeometry.geomFromString(JtsGeometry.java:108)
	at net.postgis.jdbc.jts.JtsGeometry.setValue(JtsGeometry.java:83)
	at org.postgresql.jdbc.PgConnection.getObject(PgConnection.java:654)
	at org.postgresql.jdbc.PgResultSet.getObject(PgResultSet.java:3714)
```
Example query:
`select ST_GeomFromText('POLYGON EMPTY')`

**Comment by lukaszm89** [comment]
I created PR #119

**Comment by phillipross** [comment]
fixed via PR #119

Thanks again for the contribution!
